packages/test/test-end-to-end-tests/src/test/gc/gcReferenceUpdatesInSummary.spec.ts

```diff
@@ -4,75 +4,72 @@
  */
 
 import { strict as assert } from "assert";
-
-import {
-	ContainerRuntimeFactoryWithDefaultDataStore,
-	DataObject,
-	DataObjectFactory,
-} from "@fluidframework/aqueduct";
 import { IContainer } from "@fluidframework/container-definitions";
 import { ContainerRuntime, IContainerRuntimeOptions } from "@fluidframework/container-runtime";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
-import { SharedMatrix } from "@fluidframework/matrix";
+import type { SharedMatrix } from "@fluidframework/matrix";
```

Comment:
what does this "type" do?

Reply:
It's a typescript notion that only allows referencing the symbol as a type (not a module).

I did this since:
- the DataObject that the test references `SharedTree` and `SharedMatrix` types
- we want the runtime values for SharedTree/SharedMatrix to come from the compat config
- i don't want to shadow

We could technically avoid importing the types completely by changing the field on TestDataObject to

```typescript
    	public matrix!: typeof SharedMatrix;
```

but that's a bit uglier IMO.